web/src/components/views/CreateProjectByPattern.jsx

```diff
@@ -261,15 +267,12 @@ export default class UploadByPattern extends React.Component {
           default: this.state.status
         },
         url: {
-          type: "string",
-          title: "Github URL",
-          format: "uri"
-        },
-        url_two: {
-          type: "string",
-          title: "Other URL",
-          format: "uri",
-          default: this.state.url
+          type: "array",
+          title: "URL",
+          items: {
+            type: "string"
+          },
```

Comment:
This will need to be extended soon, as we promised the alternative of adding descriptions to the links. Which means this can also be an object with the keys "title" and "url". Maybe you wanna fix that now?

Anyway this is not critical for merge.